docs/Features/Vanity.md

```diff
@@ -0,0 +1,34 @@
+# Vanity
+Satisfy all of your cosplay needs with easy-to-use, non-intrusive vanity system.
+Access its menu by right-clicking any of your equipped items and choosing "Vanity..." option.
+![Vanity submenu](img/vanity/submenu.png)
+
+From here you will have access to a number of tabs packed with customization features.
+![Tabs overview](img/vanity/tabs.png)
+
+## Transmog tab ![Transmog tab icon](img/vanity/transmog_tab_icon.png)
+![Transmog tab showcase](img/vanity/transmog_tab.png)
+Equipment tab lets you seamlessly transmogrify the appearance of your weapons and armors.  
+The item you've used to access this tab is its focus, indicated by the dropdown list at the top. To select another item to tinker with you can use rightclick->Vanity... on it or said dropdown list.  
```

Comment:
Should be "right-click" (with a dash) for consistency. I also always put a space inbetween arrows (ex. "right-click -> Vanity...")